### PR TITLE
Cleaner look for lt_radec

### PR DESCRIPTION
**Pull request by profxj**
As advertised.

Small changes for a somewhat cleaner look.

### CODE REVIEW

File: linetools/scripts/lt_radec.py [hunk 1]
```diff
@@ -43,10 +43,15 @@ def main(args=None):
     coord = ltu.radec_to_coord(icoord)
 
     # Time to print
+    print('===================================')
     print('J{:s}{:s}'.format(coord.ra.to_string(unit=u.hour,sep='',pad=True),
```
Comment (ntejos) [inline linetools/scripts/lt_radec.py]: I would remove this "===== ..." header, as no every terminal has monospace fonts.

File: linetools/scripts/lt_radec.py [hunk 2]
```diff
@@ -43,10 +43,15 @@ def main(args=None):
...
                              coord.dec.to_string(sep='',pad=True,alwayssign=True)))
+    print('   ')
+    print('   {:s} {:s}'.format(coord.ra.to_string(unit=u.hour,sep=':',pad=True),
+                             coord.dec.to_string(sep=':',pad=True,alwayssign=True)))
```
Comment (ntejos) [inline linetools/scripts/lt_radec.py]: Say explicitly that these are J2000 coordinates somewhere?

File: linetools/scripts/lt_radec.py (hunk 1 above)
Comment (profxj) [reply linetools/scripts/lt_radec.py]: changed to a blank line (helps to have that separation)

File: linetools/scripts/lt_radec.py (hunk 2 above)
Comment (profxj) [reply linetools/scripts/lt_radec.py]: added.  good suggestion.

**Comment by profxj** [comment]
merging

**Pull request merged**